Solaris specific patch to add syslog compatibility.
Not for community.


--- rsyslog-8.38.0/runtime/rsyslog.h	2018-10-19 06:16:42.055580670 +0000
+++ rsyslog-8.38.0/runtime/rsyslog.h	2018-10-19 06:15:59.929451489 +0000
@@ -161,6 +161,8 @@ extern int src_exists;
 #define	LOG_CRON	(9<<3)	/* clock daemon */
 #define	LOG_AUTHPRIV	(10<<3)	/* security/authorization messages (private) */
 #define	LOG_FTP		(11<<3)	/* ftp daemon */
+#define	LOG_AUDIT	(13<<3)	/* rfc5424 */
+#define	LOG_CRON2	(15<<3)	/* rfc5424 */
 #if defined(_AIX)		/* AIXPORT : These are necessary for AIX */
 #define	LOG_ASO		(12<<3) /* Active System Optimizer. Reserved for internal use */
 #define	LOG_CAA		(15<<3) /* Cluster aware AIX subsystem */
